code/modules/mob/new_player/sprite_accessories/ipc/ipc_face.dm

```diff
@@ -133,15 +141,3 @@
 
 /datum/sprite_accessory/hair/ipc/fluff
 	fluff = 1
```

Comment:
```suggestion
```
Guess we can remove this too since there are no other fluff ipc screens and very unlikely to have any in the future.